en/docs/api-gateway/observability/tracing/configuration.md

```
@@ -0,0 +1,197 @@
+[tracing]
+batch_timeout = "5s"            # Wait up to 5s before exporting
+max_export_batch_size = 1024    # Export up to 1024 spans per batch
+```
+
+**Lower timeout**: Faster trace visibility, more network overhead
+**Higher timeout**: Better batching efficiency, slower trace visibility
+
+## Customizing OpenTelemetry Collector
+
+The OTLP Collector configuration is located at:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)